src/lib/tokens.ts

```diff
@@ -1,4 +1,21 @@
 import { unsafeCSS } from 'lit';
 
 export const mobileBreakpoint = unsafeCSS`700px`;
-export const mobileXlBreakpoint = unsafeCSS`1008px`;
\ No newline at end of file
+export const mobileXlBreakpoint = unsafeCSS`1008px`;
```

Comment:
starting to ease into the concept of design tokens / css in JS.

Reply:
definitely!   Off-topic, but when I saw this started thinking about how we store/access those tokens in different units, maybe having conversion methods between units of measure.  Or say if you wanted the raw value `700` instead of `700px`.

Reply:
Like if the user wanted just the raw value for the breakpoint or do you mean something else?

Reply:
yes, for sure @zeroedin.  Totally agree.  I think we'll probably have a long discussion about this when we set up the design tokens repo.

Reply:
Gotcha, that makes sense. Perhaps we keep the numbers separate from the unit of measurement and add them afterwards / default to px maybe but allow the user to use the raw numbers if they'd like.